subsys/bluetooth/audio/media_control_internal.h

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
```

Comment:
Is it Github messing up the diff, or did you add this as a new file alongside the current `media_proxy_internal.h` while still keeping the original `media_proxy_internal.h` in the tree? Doesn't look it is being removed or renamed.